pkgs/development/node-packages/node-packages-v10.json

```diff
@@ -1,9 +1,142 @@
 [
-  "bower"
+  "alloy"
+, "asar"
+, "azure-cli"
+, "azure-functions-core-tools"
+, "bower"
+, "bower2nix"
+, "browserify"
+, "castnow"
+, "clean-css"
 , "coffee-script"
+, "coinmon"
+, "configurable-http-proxy"
+, "cordova"
+, "create-cycle-app"
+, "create-react-app"
+, "create-react-native-app"
+, "csslint"
+, "dat"
+, "dhcp"
+, "dnschain"
```

Comment:
That one seems inactive, I will remove it: https://github.com/nylira/prism-break/issues/1274

Reply:
@ts468 are you still using this?

Reply:
any opinions regarding that? @rnhmjoj